src/http.ts

```diff
@@ -31,12 +31,14 @@ class Http {
     path,
     data,
     host,
-    headers,
+    versioned = true,
+    headers = {},
   }: {
     method: HttpMethod;
     path: string;
     data?: any;
     host?: string;
+    versioned?: boolean;
```

Comment:
I think instead of `versioned` as a boolean, we can maybe accept `version` as an optional string here. 

In this case, the version is actually `maps`.

So below, this can take precedence if it's set in the request
```js
const urlHost = host || options.host;
const urlVersion = version || options.version;
let url = `${urlHost}/${version}/${path}`;
```